src/assets/scripts/client/parsers/aircraftCommandParser/argumentValidators.js

```diff
@@ -168,6 +168,23 @@ export const fixValidator = (args = []) => {
     }
 };
 
+/**
+ * Returns true if the value is exactly a 3 digit decimal number
+ *
+ * @function isValidCourseString
+ * @param value {string}
+ * @returns {boolean}
+ */
+export const isValidCourseString = (value) => {
+    // Can't rely on parseInt/convertStringToNumber as it'll parse 1min => 1
+    if (!/^[0-9]{3}$/.test(value)) {
```

Comment:
You regex wizards blow my mind understanding this garbage...

That said, please move it to the REGEX const so the rest of us lemmings can reuse your wizardry.

Reply:
Good point